guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,455 @@
+# How to setup Akka cluster on Kubernetes
+
+## The problem
+
+You've created an application using [Akka](http://akka.io/) with [Akka Clustering](http://doc.akka.io/docs/akka/current/scala/common/cluster.html) enabled. After evaluating all of your deployment options, you've chosen to deploy to [Kubernetes](https://kubernetes.io/) to leverage the facilities it provides for automated deployment, scaling, and management of containerized applications. This raises a problem though: how do you setup an Akka cluster on Kubernetes? Finally, running an application on Kubernetes requires containerization. How exactly do we deploy these systems to Kubernetes?
```

Comment:
Can we add some sentences about why Akka clustering services are special, e.g. forming, joining a cluster, seed nodes.